requirements.txt

```diff
@@ -22,7 +22,8 @@ marshmallow-enum==1.5.1
 Pillow==8.3.2
 sncosmo==2.5.0
 tdtax==0.1.3
-healpix-alchemy==0.1.2
+mocpy==0.8.5
```

Comment:
where is this imported or it a "must be pinned" dependency of something else?

Reply:
I am actually thinking here that we should wait for conesearch-alchemy to get dropped in on master and then rebase.

Reply:
This has been resolved.